styles/colors-silicon-day.less

```diff
@@ -266,7 +266,9 @@
 
 // IncrementSlider
 // ---------------------------------------
-@agate-incrementslider-button-color: inherit;
+@agate-incrementslider-button-bg-color: @agate-white;
+@agate-incrementslider-button-color: @agate-button-color;
+@agate-incrementslider-color: @agate-black;
```

Comment:
What do you think about setting these to the more generalized variables?

in `base`:
```
@agate-incrementslider-button-bg-color: @agate-button-bg-color;
@agate-incrementslider-button-color: @agate-button-color;
```
In silicon day (here)
```
-delete button-bg-color
-delete button-color
@agate-incrementslider-color: @agate-foreground;
```

Reply:
good idea